app/screens/home/channel_list/channel_list.tsx

```diff
@@ -111,6 +116,32 @@ const ChannelListScreen = (props: ChannelProps) => {
         return () => back.remove();
     }, [handleBackPress]);
 
+    useEffect(() => {
+        const activeServerUrlObserver = async (servers: ServersModel[]) => {
+            const server = servers.reduce((a, b) => (b.lastActiveAt > a.lastActiveAt ? b : a));
+
+            if (server) {
+                const database = DatabaseManager.serverDatabases[server.url]?.database;
+                if (database) {
+                    const userContext = await getUserContext(database);
+                    Sentry.setContext('User-Information', userContext);
+
+                    const buildContext = await getBuildTags(database);
+                    Sentry.setContext('App-Build Information', buildContext);
+
+                    const extraContext = await getExtraContext(database);
+                    Sentry.setContext('Server-Information', extraContext);
+                }
+            }
+        };
+
+        const subscription = subscribeActiveServers(activeServerUrlObserver);
+
+        return () => {
+            subscription?.unsubscribe();
+        };
+    }, []);
```

Comment:
this does not look like the right place for this, and if it is then you have access to the `serverUrl` with `useServerUrl` removing the need for the activeServerUrlObserver and subscribeActiveServers.

maybe this makes more sense to have at `home/index.tsx` ?

Reply:
moved to home.index